changelog/next/breaking-changes/3365--read-write-into-parse-print.md

```diff
@@ -0,0 +1,3 @@
+The `parse` and `print` operators have been renamed to `read` and `write`,
+respectively. The operators previously named `read` and `write` do not exist
+anymore.
```

Comment:
Actually, I changed this to:
~~~
The `parse` and `print` operators have been renamed to `read` and `write`,
respectively. The `read ... [from ...]` and `write ... [to ...]` operators
are not available anymore. If you did not specify a connector, you can
continue using `read ...` and `write ...` in many cases. Otherwise, use
`from ... [read ...]` and `to ... [write ...]` instead.
~~~

Reply:
Yes, that works for me!